bin/ghe-host-check

```diff
@@ -113,7 +113,7 @@ fi
 
 # backup-utils 2.13 onwards limits support to the current and previous two releases
 # of GitHub Enterprise Server.
-supported_minimum_version="2.17.0"
+supported_minimum_version="2.18.0"
```

Comment:
Should this be 2.20?

Reply:
Nope. We support N-2 releases like we do for GHES upgrades as we need to support migrations.